<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="filterCombo">
       <item>
        <property name="text">
         <string>Wszystkie</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dostępne</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wypożyczone</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="bookTable"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="borrowButton">
         <property name="text">
          <string>Wypożycz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="returnButton">
         <property name="text">
          <string>Zwróć</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
